spec/passaporte_web/service_account_spec.rb

```diff
@@ -43,39 +43,39 @@
       accounts_and_meta = PassaporteWeb::ServiceAccount.find_all
 
       service_accounts = accounts_and_meta.service_accounts
-      service_accounts.should be_instance_of(Array)
-      service_accounts.size.should == 15
-      service_accounts.map { |a| a.instance_of?(PassaporteWeb::ServiceAccount) }.uniq.should == [true]
-      service_accounts.map(&:plan_slug).uniq.sort.should == ['free', 'passaporteweb-client-ruby']
-      service_accounts.map(&:persisted?).uniq.sort.should == [true]
+      expect(service_accounts).to be_instance_of(Array)
+      expect(service_accounts.size).to eq(15)
+      expect(service_accounts.map { |a| a.instance_of?(PassaporteWeb::ServiceAccount) }.uniq).to eq([true])
+      expect(service_accounts.map(&:plan_slug).uniq.sort).to eq(['free', 'passaporteweb-client-ruby'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.